src/Compilers/Test/Core/AssemblyLoadTestFixture.cs

```diff
@@ -19,12 +20,34 @@ public sealed class AssemblyLoadTestFixture : IDisposable
         private readonly TempRoot _temp;
         private readonly TempDirectory _directory;
 
+        /// <summary>
+        /// An assembly with no references, assembly version 1.
+        /// </summary>
         public TempFile Delta1 { get; }
+
+        /// <summary>
+        /// An assembly with a reference to <see cref="Delta1"/>.
+        /// </summary>
         public TempFile Gamma { get; }
+
+        /// <summary>
+        /// An assembly with a reference to <see cref="Beta"/>.
```

Comment:
I think this comment should say "Gamma".

Reply:
Correct, fixed.